### DISCUSSION TITLE
static asset html file can't be visit at first time , refresh ok

### DISCUSSION THREAD

**Discussion by cocobase** [facebook/docusaurus#7426] [General] [answered]
I put my static html file in asset directory, such as [static]

I write a link in md(markdown) file to the static html file

First, when I click the url, it shows "Page not found"
![截屏2022-05-15 下午11 11 37](https://user-images.githubusercontent.com/60334709/168479987-7a7a80e4-8039-4747-887f-01deb09185c0.png)

But, when I refresh it again in browser, I works

![截屏2022-05-15 下午11 12 08](https://user-images.githubusercontent.com/60334709/168480012-acb5c75f-f465-4074-bebd-d523556437fa.png)

How to fix this problem?

**Comment by Josh-Cena** [comment, marked as answer]
Please see https://docusaurus.io/docs/advanced/routing#escaping-from-spa-redirects

**Reply by cocobase** [reply]
Thank you very much for your advise. According your suggestion document, I put pathname:// before link.
It works.
Thank you very much!
